backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -35,6 +35,9 @@ package object backend {
                                   key: BackendJobDescriptorKey,
                                   inputs: Map[LocallyQualifiedName, WdlValue]) {
     val call = key.call
+
+    /** The abort functionality potentially becomes available after the job descriptor is created */
+    var abortFunction: Option[() => Unit] = None
```

Comment:
+1 to Khalid's suggestion.

I was thinking a `var` would be used to capture the run ID (or a `Run` that encapsulates the run ID) when that becomes available, but yeah I guess you could use the metadata service for this.  Or was capturing things like run IDs supposed to be the responsibility of the KV store?